apps/docs/src/components/Downloads/PresentationMode.tsx

```
@@ -0,0 +1,561 @@
+  onClose,
+  startSlide = 0,
+}: PresentationModeProps): React.ReactElement | null {
+  const [currentSlide, setCurrentSlide] = React.useState(startSlide);
+  const [showNotes, setShowNotes] = React.useState(false);
+  const [showThumbnails, setShowThumbnails] = React.useState(false);
+  const [elapsedTime, setElapsedTime] = React.useState(0);
+  const [isTimerRunning, setIsTimerRunning] = React.useState(false);
+  const [revealedBullets, setRevealedBullets] = React.useState<number>(999); // Show all by default
+  const [animationMode, setAnimationMode] = React.useState(false);
+
+  const timerRef = React.useRef<NodeJS.Timeout | null>(null);
```

ESLint

[error] 'NodeJS' is not defined.

(no-undef)